mirage/config.js

```diff
@@ -3,6 +3,7 @@ import { Response } from 'mirage-server';
 
 export default function configure() {
   this.urlPrefix = okapi.url;
+  this.timing = 1000;
```

Comment:
should this remain here?  seems like we don't want this in top-level config, no?

Reply:
You right. Slipped in by mistake